src/main/java/org/opentripplanner/street/model/edge/StreetEdge.java

```diff
@@ -235,6 +235,9 @@ public boolean canTraverse(TraverseModeSet modes) {
    * It is used in {@link #canTraverse(TraverseMode)}
    */
   public boolean canTraverse(TraverseMode mode) {
+    if (mode.isCycling() && isStairs()) {
+      return false;
+    }
```

Comment:
Can we set this already in the WayPropetySet, then we wouldn't need to check it during runtime?

Reply:
Yes, this can go away. The tag mapper already does that (https://github.com/opentripplanner/OpenTripPlanner/blob/ad9f9512135ea1e8ec5e1d55137af3af2df5c0ed/src/main/java/org/opentripplanner/graph_builder/module/osm/tagmapping/DefaultMapper.java#L62) but my test doesn't reflect this.

Reply:
Done